core/camel-core-osgi/pom.xml

```diff
@@ -59,21 +60,21 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
-                <version>5.1.9</version>
+                <version>${maven-bundle-plugin-version}</version>
                 <extensions>true</extensions>
                 <inherited>true</inherited>
                 <configuration>
                     <instructions>
                         <Export-Package>
-                            org.apache.camel*
```

Comment:
That's a fair point. The reason why I put this was because the first version wrapped all core artifacts in one uber bundle.
However, as we now have individual wrappers for core, it makes sense.